starter-s3.yaml

```diff
@@ -143,13 +143,75 @@ data:
     presubmits:
 
     periodics:
-    - interval: 20m
+    - interval: 3h
```

Comment:
I remember I asked you to update the internal in the past. Did you forget updating the yaml last time..? 
![image](https://user-images.githubusercontent.com/4739316/162759044-a826345f-f715-4de8-aed2-5143e94e7b2b.png)

Reply:
Yes...my bad...I forgot to sent a PR last time instead updated the online config.